web/src/screens/filters/details.tsx

```diff
@@ -368,28 +371,31 @@ export function MoviesTv() {
   return (
     <div>
       <div className="mt-6 grid grid-cols-12 gap-6">
-        <TextField name="shows" label="Movies / Shows" columns={8} placeholder="eg. Movie,Show 1,Show?2" />
+        <TextFieldIcon name="shows" label="Movies / Shows" columns={8} placeholder="eg. Movie,Show 1,Show?2" id="shows"/>
+        <Tooltip style= {{ fontSize: "12px", borderRadius: "0.375rem", backgroundColor: "#18181B", color: "#fff" }} place="right" anchorId="shows" data-html={true} clickable={true} html="<div><p>You can use basic filtering like wildcards (*) or replace single characters with <code>?</code>.</p><a href='https://autobrr.com/filters#tvmovies' class='text-blue-400 visited:text-blue-400' target='_blank'>https://autobrr.com/filters#tvmovies</a></div>"/>
+        
```

Comment:
With the proposed changes to the original `TextField` and other components to take a `tooltip` component, you can then make some changes here.

Make a new component somewhere in components dir that wraps this current `<Tooltip />` component looking something like this:

```typescript
interface CustomTooltipProps {
  anchorId: string;
  children: React.ReactNode;
  clickable?: boolean;
}

export const CustomTooltip = ({
  anchorId,
  children,
  clickable
}: CustomTooltipProps) => (
  <>
  <svg className="float-right ml-1 -mt-1 h-5 w-5 text-gray-500" width="800px" height="800px" viewBox="0 0 1024 1024" xmlns="http://www.w3.org/2000/svg">
          <path fill="#333" d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"/>
          <path fill="#E6E6E6" d="M512 140c-205.4 0-372 166.6-372 372s166.6 372 372 372 372-166.6 372-372-166.6-372-372-372zm32 588c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V456c0-4.4 3.6-8 8-8h48c4.4 0 8 3.6 8 8v272zm-32-344a48.01 48.01 0 0 1 0-96 48.01 48.01 0 0 1 0 96z"/>
          <path fill="#333" d="M464 336a48 48 0 1 0 96 0 48 48 0 1 0-96 0zm72 112h-48c-4.4 0-8 3.6-8 8v272c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V456c0-4.4-3.6-8-8-8z"/>
  </svg>
  <Tooltip style= {{ fontSize: "12px", borderRadius: "0.375rem", backgroundColor: "#18181B", color: "#fff" }} place="right" anchorId={anchorId} data-html={true} clickable={clickable}>
    {children}
  <Tooltip />
 </>
)
```

Then you can call `TextField` like

```typescript
<TextField name="shows" label="Movies / Shows" columns={8} placeholder="eg. Movie,Show 1,Show?2" id="shows" tooltip={<div><p>You can use basic filtering like wildcards (*) or replace single characters with <code>?</code>.</p><a href='https://autobrr.com/filters#tvmovies' className='text-blue-400 visited:text-blue-400' target='_blank'>https://autobrr.com/filters#tvmovies</a></div>} />
```

And voila you have a reusable component now!